apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/store.js

```diff
@@ -0,0 +1,35 @@
+import { createRegistrySelector, registerStore } from '@wordpress/data';
+
+const DEFAULT_STATE = {
+	isModalVisible: true,
+};
+
+registerStore( 'automattic/wpcom-global-styles', {
+	reducer: ( state = DEFAULT_STATE, action ) => {
+		switch ( action.type ) {
+			case 'DISMISS_MODAL':
+				return {
+					...state,
+					isModalVisible: false,
+				};
```

Comment:
No need to change this.
Couldn't we just `return { isModalVisible: false }` in this case since the state consists of only one property?

Reply:
Yup, that'd be possible. We could also remove the switch statement, since we only have one action right now. But that would leave the reducer on a non-scalable mode (which might be ok since we may never need to scale it) and  it'd feel a bit strange to folks used to how reducers are maintained, since most of them (including simple ones) are implemented like this. So I decided to stick with the usual format in order to keep things as expected, considering that this format doesn't add too much complexity for our use-case.